flow/project.py

```diff
@@ -2336,20 +2336,24 @@ def compute_status(data):
                     selected_groups=single_operation_groups,
                 )
             )
-            # aggregate_groups is a list of tuples containing scheduler,
-            # aggregate, and group information. To compute labels, we fetch the
-            # unique jobs from the aggregates containing only one job.
-            individual_jobs = {
-                aggregate_group[3][0]
-                for aggregate_group in aggregate_groups
-                if len(aggregate_group[3]) == 1
-            }
             status_results = parallel_executor(
                 compute_status,
                 aggregate_groups,
                 desc="Fetching status",
                 file=err,
             )
+            # aggregate_groups is a list of tuples containing scheduler,
+            # aggregate, and group information. To compute labels, we fetch the
+            # unique jobs from the aggregates containing only one job.
+            if len(single_operation_groups) > 0:
+                individual_jobs = {
+                    aggregate_group[3][0]
+                    for aggregate_group in aggregate_groups
+                    if len(aggregate_group[3]) == 1
+                }
+            else:
+                # If no operations exist, use all jobs in the project.
+                individual_jobs = set(self)
```

Comment:
Why did the location of this block move? This change doesn't seem necessary.

Reply:
@bdice any comments?

Reply:
I just wanted to move the construction of `individual_jobs` closer to the place where it's actually used.